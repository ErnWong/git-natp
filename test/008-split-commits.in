A1---A2---A3---BB12--BB23--BB34--BB4 master
      \      .~'/  .~'/  .~'/  .~'
       \  .~'  /.~'  /.~'  /.~'
        B1----B2----B3----B4
