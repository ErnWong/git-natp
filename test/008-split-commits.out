A1
A2 A1
B1 A2
A3 A2
B2 B1
BB12 A3 B1 B2
B3 B2
BB23 BB12 B2 B3
B4 B3
BB34 BB23 B3 B4
BB4 BB34 B4
[master] BB4
